src/js/components/TaskDetail.js

```diff
@@ -27,14 +32,14 @@ const METHODS_TO_BIND = [
   'onTaskDirectoryStoreSuccess'
 ];
 
-class TaskSidePanelContents extends SidePanelContents {
+class TaskDetail extends mixin(InternalStorageMixin, TabsMixin, StoreMixin) {
   constructor() {
     super(...arguments);
 
     this.tabs_tabs = Object.assign({}, TABS);
 
     this.state = {
-      currentTab: Object.keys(this.tabs_tabs).shift(),
+      currentTab: 'details',
```

Comment:
This should to be the first item in the `this.tabs_tabs` array so we don't rely on manual configuration. See comment https://github.com/dcos/dcos-ui/pull/118/files#r62645384